build/package/rpm/go-fdo-server.spec

```diff
@@ -44,6 +46,12 @@ BuildRequires:  systemd-rpm-macros
 %global gomodulesmode GO111MODULE=on
 %gobuild -o %{gobuilddir}/bin/go-fdo-server %{goipath}
 
+# Strip debug symbols
+%if %{defined _hardened_build}
+%global _hardened_build 1
+%endif
+strip %{gobuilddir}/bin/go-fdo-server
```

Comment:
Do this in the %install section when installing the server binary with the `-s` flag:

```
install -m 0755 -vp -s -D -t %{buildroot}%{_bindir}/ %{gobuilddir}/bin/*
```